src/commands/info/infoHandler.ts

```diff
@@ -0,0 +1,28 @@
+import { InfoOptions } from 'src/types';
+import { downloadFromIPFS } from '~/lib/ipfs';
+import { networkSelectionPrompt, pluginSelectionPrompt } from '~/lib/prompts';
+import { findNetworkByName } from '~/lib/web3';
+import { logTable, strings } from '~/lib/constants';
+import { findRepo, queryPluginRepos } from '~/lib/subgraph';
+import { validateNetwork, validateSubdomain } from '~/lib/validators';
+
+export const infoHandler: (...args: any[]) => void | Promise<void> = async (
+  repoName?: string,
+  options: InfoOptions = {},
+) => {
+  validateSubdomain(repoName);
+  validateNetwork(options.network);
+
+  const network = options.network ? findNetworkByName(options.network) : await networkSelectionPrompt();
+  const pluginRepos = await queryPluginRepos(network.subgraph);
+  const plugin = repoName ? findRepo(repoName, pluginRepos) : await pluginSelectionPrompt(pluginRepos);
+
+  const releaseMetadata = await downloadFromIPFS(plugin.releases[0].metadata);
+
+  logTable([
+    { NAME: releaseMetadata.name || strings.NO_NAME_PROVIDED },
+    { SUBDOMAIN: plugin.subdomain },
+    { DESCRIPTION: releaseMetadata.description || strings.NO_DESCRIPTION_PROVIDED },
+    { ADDRESS: plugin.id },
```

Comment:
Shall we add maintainer here as well?

Reply:
yes we should! The problem is the maintainer/s are not being indexed in the subgraph and AFAIK you cant query the `PermissionManager` for a list of addresses with a particular permission (unless im missing something). So we can do it but its alot of work

Reply:
ok cooool. let's leave it as is for now and chat with the OS team to provide this on their end.